build-binary-template.yml

```diff
@@ -31,6 +31,14 @@ jobs:
           shopt -s extglob
           mv !(gopath) '$(modulePath)'
           echo '##vso[task.prependpath]$(GOBIN)'
+          curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh| sh -s -- -b '$(GOPATH)/bin' v1.21.0
```

Comment:
I think this command failed on windows, but I have no access to check the actual build log.

Reply:
We'll get build logs sorted out asap

Reply:
thanks 💯, i will try to wrap it up once having access to build log later :)